src/translations/sqlb_de.ts

```diff
@@ -1165,7 +1165,7 @@ Fehler werden mittels eine roten Wellenlinie angezeigt.</translation>
     <message>
         <location filename="../EditDialog.ui" line="295"/>
         <source>Ctrl+Shift+C</source>
-        <translation></translation>
+        <translation>Strg+Umschalt+C</translation>
```

Comment:
```suggestion
        <translation></translation>
```

Keyboard shortcuts get automatically localized by Qt, and translating key names may indeed break them. They show up on the translation file only for the rare occasions that one has to redirect a shortcut to another one in a specific language. Further information: https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translations#dont-translate-key-names-in-shortcuts